### ISSUE TITLE
Multi language

### ISSUE THREAD

**Issue by jmverges** [ahadb/flat-file-cms#74]
Hi there,

Could the list inform if the CMSs are capable of multilanguage?

Cheers.

**Comment by mciszczon** [comment]
So you probably mean support for multilang **by default**? That would be cool.

**Comment by jmverges** [comment]
@mciszczon exactly that

**Comment by filisko** [comment]
Yep, I think that would be useful

**Comment by ahadb** [comment]
Indeed, that would be useful - also if anyone would be interested in translating the readme to another language, just drop me an email. Thanks
